### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by victor-paumier**

### CODE REVIEW

File: assets/js/tools/tools.js [hunk 1]
```diff
@@ -60,9 +60,10 @@ class tools {
      */
     static initDatatable(elem, options) {
         let params = {
-            scrollX: true,
+            // scrollX: true,
```
Comment (JeremieSamson) [inline assets/js/tools/tools.js]: Remove comment here

File: assets/sass/main.css [hunk 2]
```diff
@@ -78,7 +78,8 @@
   height: 180px;
   transform: skew(0, -7deg);
   transform-origin: 0 0;
-  background-color: #f8f9fd; }
+  background-color: #f8f9fd;
```
Comment (JeremieSamson) [inline assets/sass/main.css]: maybe we should put `#f8f9fd` into a variable ?

File: assets/sass/main.scss [hunk 3]
```diff
@@ -386,4 +528,30 @@ span.select2 {
...
+  &.matching_list, &#sections,#buddies {
+    min-width: 750px;
+  }
+  thead {
+    .sorting, .sorting_asc {
+      font-size: 12px;
+      vertical-align: middle;
+    }
+  }
 }
```
Comment (JeremieSamson) [inline assets/sass/main.scss]: cs => missing line break

File: src/DataFixtures/BuddyFixtures.php [hunk 4]
```diff
@@ -15,7 +15,7 @@ class BuddyFixtures extends Fixture implements OrderedFixtureInterface
     public function load(ObjectManager $manager)
     {
 
-        for ($i = 0; $i < UserFixtures::NB_USER_TO_CREATE; ++$i) {
+        for ($i = 0; $i < (UserFixtures::NB_USER_TO_CREATE / 2); ++$i) {
```
Comment (JeremieSamson) [inline src/DataFixtures/BuddyFixtures.php]: cs

File: src/DataFixtures/UserFixtures.php [hunk 5]
```diff
@@ -47,11 +48,18 @@ public function load(ObjectManager $manager)
 
         $manager->persist($bc);
 
+        /** @var Language $lnEn */
+        $lnEn = $this->getReference(LanguageFixtures::LANGUAGE_REFERENCE.'_0');
+
+        /** @var Language $lnFr */
+        $lnFr = $this->getReference(LanguageFixtures::LANGUAGE_REFERENCE.'_1');
+
         for ($i = 0; $i < self::NB_USER_TO_CREATE; ++$i) {
```
Comment (JeremieSamson) [inline src/DataFixtures/UserFixtures.php]: cs

File: templates/Buddy/myBuddies.html.twig [hunk 6]
```diff
@@ -2,39 +2,48 @@
...
-        <tbody>
-        {% for buddy in buddies %}
-        {% if user == buddy.mentor or user == buddy.mentee %}
-
-        {% if buddy.mentor.section is defined %}
-                {% if buddy.mentor.section == section %}
+    <div class="container mt-5 mb-5 white-box p-3">
+        <div class="row">
+            <div class="col">
+                <h1>Buddies list</h1>
```
Comment (JeremieSamson) [inline templates/Buddy/myBuddies.html.twig]: Translation

File: templates/Buddy/myBuddies.html.twig [hunk 7]
```diff
@@ -2,39 +2,48 @@
...
+            <div class="col">
+                <table>
+                    <thead>
                     <tr>
-                        <td><a href="{{ path('buddy_show', { 'id': buddy.id }) }}">{{ buddy.id }}</a></td>
-                        <td>{% if buddy.created %}{{ buddy.created|date('Y-m-d H:i:s') }}{% endif %}</td>
-                        <td>{{ buddy.confirmed }}</td>
-                        <td>{{ buddy.refusal }}</td>
-                        <td>{% if buddy.responseAt %}{{ buddy.responseAt|date('Y-m-d H:i:s') }}{% endif %}</td>
+                        <th>Id</th>
```
Comment (JeremieSamson) [inline templates/Buddy/myBuddies.html.twig]: Those too

File: templates/Buddy/myBuddies.html.twig [hunk 8]
```diff
@@ -2,39 +2,48 @@
...
+                        <th>Actions</th>
                     </tr>
-                {% endif %}
-            {% endif %}
-        {% endif %}
+                    </thead>
+                    <tbody>
+                    {% for buddy in buddies %}
+                        {% if user == buddy.mentor or user == buddy.mentee %}
+
```
Comment (JeremieSamson) [inline templates/Buddy/myBuddies.html.twig]: line break not needed

File: templates/Buddy/myBuddies.html.twig [hunk 9]
```diff
@@ -2,39 +2,48 @@
...
+                                        <td><a href="{{ path('buddy_show', { 'id': buddy.id }) }}">{{ buddy.id }}</a></td>
+                                        <td>{% if buddy.created %}{{ buddy.created|date('Y-m-d H:i:s') }}{% endif %}</td>
+                                        <td>{{ buddy.confirmed }}</td>
+                                        <td>{{ buddy.refusal }}</td>
+                                        <td>{% if buddy.responseAt %}{{ buddy.responseAt|date('Y-m-d H:i:s') }}{% endif %}</td>
+                                    </tr>
+                                {% endif %}
+                            {% endif %}
+                        {% endif %}
+
```
Comment (JeremieSamson) [inline templates/Buddy/myBuddies.html.twig]: same here

File: templates/Home/partners.html.twig [hunk 10]
```diff
@@ -94,7 +119,7 @@
     </div>
 
 
-    <div class="grey-background pb-5">
+    <div class="grey-background pb-5 ">
```
Comment (JeremieSamson) [inline templates/Home/partners.html.twig]: remove last space

File: templates/Section/index.html.twig [hunk 11]
```diff
@@ -1,54 +1,66 @@
 {% extends 'base.html.twig' %}
 
 {% block body %}
-    <div class="container bg-white">
-        <h1>Section list</h1>
+    <div class="container bg-white white-box p-3 mt-5 mb-5">
+        <div class="row">
+            <div class="col text-center">
+                <h1 class="f-magenta">Section list</h1>
```
Comment (JeremieSamson) [inline templates/Section/index.html.twig]: translation

File: assets/sass/main.css (hunk 2 above)
Comment (victor-paumier) [reply assets/sass/main.css]: agree

File: assets/sass/main.scss (hunk 3 above)
Comment (victor-paumier) [reply assets/sass/main.scss]: global lint ;)

Comment (victor-paumier) [reply assets/sass/main.scss]: do you use sass-lint for that??

File: templates/Section/index.html.twig (hunk 11 above)
Comment (victor-paumier) [reply templates/Section/index.html.twig]: no translations on admin only pages

File: templates/Buddy/myBuddies.html.twig (hunk 7 above)
Comment (victor-paumier) [reply templates/Buddy/myBuddies.html.twig]: didn't get it, file not used yet anyway

File: src/DataFixtures/UserFixtures.php (hunk 5 above)
Comment (victor-paumier) [reply src/DataFixtures/UserFixtures.php]: no change after cs fix

File: assets/sass/main.css (hunk 2 above)
Comment (JeremieSamson) [reply assets/sass/main.css]: is this file generated ?if yes we may not need it on the repo, or do we have a deploy constraint here ?

File: assets/sass/main.scss (hunk 3 above)
Comment (JeremieSamson) [reply assets/sass/main.scss]: nop but we should add it along with twig, yaml, php and js files

File: src/DataFixtures/UserFixtures.php (hunk 5 above)
Comment (JeremieSamson) [reply src/DataFixtures/UserFixtures.php]: though there was a space my bad ;)

File: templates/Buddy/myBuddies.html.twig (hunk 7 above)
Comment (JeremieSamson) [reply templates/Buddy/myBuddies.html.twig]: yep but we should not have plein text in twig files

File: templates/Section/index.html.twig (hunk 11 above)
Comment (JeremieSamson) [reply templates/Section/index.html.twig]: we should have translation everywhere

File: assets/sass/main.css (hunk 2 above)
Comment (victor-paumier) [reply assets/sass/main.css]: compiled file, should not be versioned

**Pull request merged**